microsoft/testsuites/network/netinterface.py

```diff
@@ -188,5 +189,30 @@ def validate_set_static_mac(self, node: Node, log: Logger) -> None:
                 node_nic_info.initialize()
 
     def _validate_netvsc_built_in(self, node: Node) -> None:
-        if node.tools[KernelConfig].is_built_in("CONFIG_HYPERV_NET"):
+        if isinstance(node.os, FreeBSD):
+            # Use command "config -x /boot/kernel/kernel | grep hyperv" can also check
+            # if netvsc is build-in. The output "device hyperv" means the the hyperv
+            # drivers includes vmbus,kvp,netvsc,storvsc are built-in to the kernel.
+            # Here use command "kldstat | grep hv_netvsc" to check.
+            is_built_in_module = (
+                node.execute(
+                    "kldstat | grep hv_netvsc", sudo=True, shell=True
+                ).exit_code
+                != 0
+            )
+        else:
+            try:
+                is_built_in_module = node.tools[KernelConfig].is_built_in(
+                    "CONFIG_HYPERV_NET"
+                )
+            except LisaException as identifier:
+                # Some image's kernel config is inconsistent with the kernel version.
+                # E.g. fatpipe-inc fatpipe-wanopt 10 0.0.3, then it has the exception.
+                # If so, check if netvsc is built-in using command "modinfo hv_netvsc".
+                node.log.debug(identifier)
+                is_built_in_module = (
+                    node.execute("modinfo hv_netvsc", sudo=True, shell=True).exit_code
+                    != 0
+                )
```

Comment:
I recall if the module is built in, modinfo also can show output, could you please double check?

Reply:
Changed to check '/lib/modules/$(uname -r)/modules.builtin'